doc/source/cluster/index.rst

```diff
@@ -3,227 +3,33 @@
-  Nmap scan report for compute04.berkeley.edu (123.456.78.910)
-  Host is up, received echo-reply ttl 60 (0.00087s latency).
-  rDNS record for 123.456.78.910: compute04.berkeley.edu
-  PORT     STATE SERVICE REASON         VERSION
-  6379/tcp open  redis   syn-ack ttl 60 Redis key-value store
-  Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
-  $ nc -vv -z $HEAD_ADDRESS $PORT
-  Connection to compute04.berkeley.edu 6379 port [tcp/*] succeeded!
-
-If the node cannot access that port at that IP address, you might see
-
-.. code-block:: bash
-
-  $ nmap -sV --reason -p $PORT $HEAD_ADDRESS
-  Nmap scan report for compute04.berkeley.edu (123.456.78.910)
-  Host is up (0.0011s latency).
-  rDNS record for 123.456.78.910: compute04.berkeley.edu
-  PORT     STATE  SERVICE REASON       VERSION
-  6379/tcp closed redis   reset ttl 60
-  Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
-  $ nc -vv -z $HEAD_ADDRESS $PORT
-  nc: connect to compute04.berkeley.edu port 6379 (tcp) failed: Connection refused
-
-
-Stopping Ray
-~~~~~~~~~~~~
-
-When you want to stop the Ray processes, run ``ray stop`` on each node.
-
-.. _using-ray-on-a-cluster:
-
-Running a Ray program on the Ray cluster
-----------------------------------------
-
-To run a distributed Ray program, you'll need to execute your program on the same machine as one of the nodes.
-
-.. tabs::
-  .. group-tab:: Python
-
-    Within your program/script, you must call ``ray.init`` and add the ``address`` parameter to ``ray.init`` (like ``ray.init(address=...)``). This causes Ray to connect to the existing cluster. For example:
-
-    .. code-block:: python
-
-        ray.init(address="auto")
-
-  .. group-tab:: Java
-
-    You need to add the ``ray.address`` parameter to your command line (like ``-Dray.address=...``).
+One of Ray's strengths is the ability to leverage multiple machines in the same program. Ray can, of course, be run on a single machine (and is done so often) but the real power is using Ray on a cluster of machines.
 
-    To connect your program to the Ray cluster, run it like this:
+A Ray cluster consists of a **head node** and a set of **worker nodes**. The head node needs to be started first, and the worker nodes are given the address of the head node to form the cluster.
 
-        .. code-block:: bash
+Distributed Ray is powered by the Ray Cluster Launcher, which automatically provisions machines and launches a multi-node Ray cluster. You can use the cluster launcher on GCP, Amazon EC2, Azure, or even Kubernetes. Distributed Ray also makes use of the Ray Autoscaler, which gives Ray the ability "auto-scale," meaning that it can interact with a Cloud Provider to request or release instances according to application workload.
 
-            java -classpath <classpath> \
-              -Dray.address=<address> \
-              <classname> <args>
+How does it work?
```

Comment:
"How does it work" should not come before "Supported Features" - can you please move this paragraph down to the bottom of the page? 

The below paragraphs are details that reference a bunch of new concepts.

Reply:
The paragraph about supported features was a bit confusing, so I removed it altogether